bindings/java/fdbJNI.cpp

```diff
@@ -134,6 +137,39 @@ void throwParamNotNull(JNIEnv* jenv) {
 	throwNamedException(jenv, "java/lang/IllegalArgumentException", "Argument cannot be null");
 }
 
+bool getBytes(JNIEnv* jenv,
```

Comment:
This `getBytes` method is particular to the `getMappedRange` APIs, right? I feel like this should be named something like `extractGetMappedRangeArgs` or something in that case.